addon/components/base/bs-form/element/errors.js

```diff
@@ -24,5 +23,12 @@ export default Component.extend({
    * @type {Ember.Array}
    * @public
    */
-  messages: null
+  messages: null,
+
+  /**
+   * @property showMultipleErrors
+   * @type {Boolean}
+   * @public
```

Comment:
Oh, sorry, overlooked this: could you please add a short description sentence, what this property does, and add `@default false`, so the automatically generated API docs are complete.

Reply:
Done.
Although the variable name is self explanatory and a commentary makes me feels like I am doing something like this:

```
class Clazz {
  /**
   * Constructs a new Clazz object.
   */
  Clazz() {
  }
}
```